### DISCUSSION TITLE
Support for offline interaction in the PWA

### DISCUSSION THREAD

**Discussion by mmnaseri** [outline/outline#2672] [General]
Right now, the PWA is online only. That means that, say, if I were on a plane and wanted to read my team meeting notes or jot down a couple of ideas, there would be no way for me to do this in Outline as of this moment.

Having this feature will greatly improve the usability of Outline and will make it much more attractive as a solution for teams.

**Comment by mmuenker** [comment]
The feature that all documents are available in offline mode is somewhat unrealistic. But a feature to mark certain documents as offline available like in Google Drive is useful.

But I think it would be useful to first allow to create new documents. So it would be possible to work on a new Idea or Documentation without an internet connection.

**Comment by mmnaseri** [comment]
Yeah, that would be a good start. I think having the ability to mark certain docs as available for offline editing is the right way to look at this.

**Reply by hilsonp** [reply]
I fully agree. 
The fact that I cannot work offline keeps me working in Notion. 
I understand that keeping all pages offline can be problematic but having a preselected set of pages editable offline and being create new ones would let me use outline for everything.

**Reply by kszinhu** [reply]
Tenho a mesma opinião, é literalmente o que me mantém no Obsidian/Notion.

**Comment by Raphmatt** [comment]
This would be a really nice feature. Working offline combined with the fact that yjs can sync back the changes would be really great for this project.
https://docs.yjs.dev/getting-started/allowing-offline-editing
